README.md

```diff
@@ -52,6 +52,31 @@ in napalm.buildPackage ./. { packageLock = <path/to/package-lock>; }
 
 If you want to use Napalm in your flake project, you can do that by adding it to your inputs and passing `napalm.overlay` to your Nixpkgs instance. There is also a template, that can help you use napalm in your project. You can use it by running `nix flake init -t "github:nix-community/napalm"`.
```

Comment:
Maybe move the sentences about template after the example?

Reply:
Thatd make sense

Reply:
Created a new section for it below the example section in b5aeb19